templates/frontend/pages/userLogin.tpl

```diff
@@ -73,6 +73,18 @@
 					</span>
 				</label>
 			</div>
+
+			{* recaptcha spam blocker *}
+			{if $recaptchaPublicKey}
+				<fieldset class="recaptcha_wrapper">
+					<div class="fields">
+						<div class="recaptcha">
+							<div class="g-recaptcha" data-sitekey="{$recaptchaPublicKey}"></div>
```

Comment:
I suggest applying the `escape` Smarty modifier to `$recaptchaPublicKey`.

Reply:
I'll replace another occurrence of it in the codebase.